spec/aptible/resource/base_spec.rb

```diff
@@ -646,4 +661,54 @@ def configure_new_coordinator(&block)
       expect(m.token).to eq('bar')
     end
   end
+
+  describe '_type' do
```

Comment:
I added this because there are some bits of the embedded hyperresource gem that I don't really understand (I have no idea what they're used for), which are probably broken in this change. However, it does look like we never use them.

Ultimately, the right approach here is going to be to move off of HyperResource. We've known that for a long time, though.

Reply:
👍